commit d1eb1eec4eea4129eb2b858820b7c2192637564f (HEAD -> master)
Author: Isaac Davis <isaac.davis@joyent.com>
Date:   2019-07-25T23:00:17+00:00 (2 months ago)
    
    MANTA-4375 port manta-create-topology.sh to node
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Reviewed by: Ryan Puckett <ryan.puckett@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/docs/operator-guide.md b/docs/operator-guide.md
index 7093c9b..d506605 100644
--- a/docs/operator-guide.md
+++ b/docs/operator-guide.md
@@ -917,7 +917,7 @@ multi-DC, multi-compute-node deployment.  The general process is:
     c. For any other installation (including a multi-CN installation), you'll
        need to run several more steps: assign shards for storage and object
        metadata with "manta-shardadm"; create a hash ring with
-       "manta-create-topology.sh"; generate a "manta-adm" configuration file
+       "manta-adm create-topology"; generate a "manta-adm" configuration file
        (see "manta-adm configuration" below); and finally run "manta-adm update
        config.json" to deploy those zones.  Your best bet is to examine the
        "manta-deploy-dev" script to see how it uses these tools.  See
